### PR TITLE
Static code analysis

### PR DESCRIPTION
**Pull request by jtanx**
This fixes a few of the newly reported Coverity warnings.

cc @frank-trampe

### CODE REVIEW

File: fontforge/cvundoes.c [hunk 1]
```diff
@@ -1426,7 +1426,7 @@ return( copy(""));
 
     for ( lcnt = ly_fore; lcnt<dummy.layer_cnt; ++lcnt )
 	RefCharsFree(dummy.layers[lcnt].refs);
-    if ( dummy.layer_cnt!=2 )
```
Comment (frank-trampe) [inline fontforge/cvundoes.c]: What is the significance of this magic number?

Comment (adrientetar) [inline fontforge/cvundoes.c]: @frank-trampe: who knows… is this a merge blocker?

Comment (frank-trampe) [inline fontforge/cvundoes.c]: It's not a major issue, but I like to try to preserve as much useful information as possible from the reverse engineering required for each change set.

**Comment by adrientetar** [comment]
@jtanx: have you got any new build for Windows coming up?

**Comment by jtanx** [comment]
@adrientetar SourceForge is still down (cannot upload new releases) and there hasn't been a new tag/release yet this month, so for now I've stuck [it](https://github.com/fontforge/fontforge/releases/download/20150612/FontForgeSetup-23-07-2015-prerelease.exe) on the [current release](https://github.com/fontforge/fontforge/releases/tag/20150612).

**Pull request merged**